packages/rexray/buildinfo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "docker": "golang:1.6",
+  "single_source" : {
+    "kind": "git",
+    "git": "git@github.com:dcos/rexray.git",
+    "ref": "1816ee3868550e07b90f00f57bfd7f737235cd93",
+    "ref_origin": "v0.3.3-dcos"
```

Comment:
we've noticed a serious regression against this branch of rexray, whereby many volumes are created when launching a Service that uses external volumes and the docker containerizer. troubleshooting this with the velocity team now.